test/common.js

```diff
@@ -21,6 +76,8 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+refreshTmpDir();
```

Comment:
I think my preference would be to make this callable by the individual tests rather than happening for _each_ test, we're trying to speed up tests and this will just slow them down